content/docs/command-reference/get-url.md

```diff
@@ -72,6 +72,10 @@ $ wget https://example.com/path/to/data.csv
 
 ## Options
 
+- `-j <number>`, `--jobs <number>` - parallelism level for DVC to download data
```

Comment:
can we do a temp mark (available since 2.0) for now? And may be actually good idea to even keep those, I don't know.

Reply:
Could but we already merged several other small changes (e.g. #2082 but others after that too I think) so we'd need to go back and detect them.

If we plan to have these permanent "since x.y" notes then that is worth it. But... That seems pretty difficult to manage manually.

Reply:
\* could be difficult to maintain manually going fwd

Reply:
yep, it'll complicate it a bit. Anyway it's a separate discussion to have. We can create a research ticket and see what we can do there.